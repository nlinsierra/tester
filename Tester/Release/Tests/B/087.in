9S 7C 5S AC 6C AS 3D KD 4D 7H 3C KC
