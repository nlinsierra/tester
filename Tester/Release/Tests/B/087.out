40
2C 4C 5C 8C 9C TC JC QC 2D 5D 6D 7D 8D 9D TD JD QD AD 2H 3H 4H 5H 6H 8H 9H TH JH QH KH AH 2S 3S 4S 6S 7S 8S TS JS QS KS
